src/Agent.php

```diff
@@ -28,19 +27,16 @@
 class Agent extends NipwaayoniAgent
 {
     protected $collectors;
-    protected $request_start_time;
 
     public function __construct(
         Config $config,
-        ContextCollection $sharedContext,
+        ContextCollection $shared_context,
         Connector $connector,
-        EventFactoryInterface $eventFactory,
-        TransactionsStore $transactionsStore,
-        RequestStartTime $startTime
```

Comment:
Just to confirm, the start time is being removed from the `Agent` class because it will now be provided directly to the `EventDataCollector` classes?

Reply:
Correct, everywhere the start time is needed, it will be injected from now on.